gcp_variant_transforms/libs/beam_util_test.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2018 Google Inc.  All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Unit tests for beam_util_test module."""
+
+from __future__ import absolute_import
+
+import unittest
+
+from apache_beam.runners import runner
+from gcp_variant_transforms.libs import beam_util
+
+_TEST_COUNTER = 'test_counter'
+_COUNTER_VALUE = 12
+
+
+# Note this is not really a good test as we are mocking objects that we
+# don't own (i.e., they are in our upstream dependency) but filling a counter
+# and connecting it to PipelineResults properly seems excessive for this test.
+class MockMetrics(object):
+  def __init__(self):
```

Comment:
nit: please add a blank link before every def (here and everywhere).

Reply:
Done.
Are you aware of a way to tell pylint to catch this? I prefer style related issues being caught by tools as much as possible.

Reply:
+1 for pylint to catch this, but I don't know how to do it. Can you please look into adding it?